test/integration/conftest.py

```diff
@@ -140,7 +141,11 @@ def db(mongo_client):
 
 @pytest.fixture(scope="class")
 def empty_db(db):
-    PRESERVED_COLLECTIONS = ["groups", "organizations", "allowedDomains"]
+    PRESERVED_COLLECTIONS = ["users", "groups", "organizations", "allowedDomains"]
+    admin_found = False
```

Comment:
`admin_found` and `admin_email` are redundant here

Reply:
Removed.